### ISSUE TITLE
deliveryservice_requests Internal Server Error when status and changeType are wrong

### ISSUE THREAD

**Issue by ZugNZwang** [apache/trafficcontrol#4395]
## I'm submitting a ...

- [X] bug report
- [ ] new feature / enhancement request
- [ ] improvement request (usability, performance, tech debt, etc.)
- [ ] other 

## Traffic Control components affected ...

- [ ] CDN in a Box
- [ ] Documentation
- [ ] Grove
- [ ] Traffic Control Client
- [ ] Traffic Monitor
- [X] Traffic Ops
- [ ] Traffic Ops ORT
- [ ] Traffic Portal
- [ ] Traffic Router
- [ ] Traffic Stats
- [ ] Traffic Vault
- [ ] unknown

## Current behavior:

When making a `GET` request to `deliveryservice_requests` with the query parameter `status` set to a status that doesn't exist the response returned is:
```json
HTTP/1.1 500 Internal Server Error
{
    "alerts": [
        {
            "text": "Internal Server Error",
            "level": "error"
        }
    ]
}
``` 
**ALSO:**
When making a `GET` request to `deliveryservice_requests` with the query parameter `changeType` set to a type that doesn't exist the response returned is:
```json
HTTP/1.1 500 Internal Server Error
{
    "alerts": [
        {
            "text": "Internal Server Error",
            "level": "error"
        }
    ]
}
``` 
Valid changeType values are: create, update, delete
## Expected / new behavior:

Should return:
```json
HTTP/1.1 200 OK
{
    "response": []
}
```
OR
```json
HTTP/1.1 400 Bad Request
{
    "alerts": [
        {
            "text": "Not a valid status",
            "level": "error"
        }
    ]
}
```

```json
HTTP/1.1 400 Bad Request
{
    "alerts": [
        {
            "text": "Not a valid changeType",
            "level": "error"
        }
    ]
}
```

**Comment by mitchell852** [comment]
@ocket8888 - think this can be closed since the ds_requests endpoint changed with api 4.x?

**Comment by ocket8888** [comment]
Not necessarily. We should try to reproduce it. If we can't, it can be closed.
